contrib/platform/src/com/sun/jna/platform/win32/Advapi32Util.java

```diff
@@ -961,6 +961,19 @@ public static String registryGetExpandableStringValue(HKEY hKey, String value) {
                 && rc != W32Errors.ERROR_INSUFFICIENT_BUFFER) {
             throw new Win32Exception(rc);
         }
+        return regMultiSzBufferToStringArray(data);
+    }
+
+    /**
+     * Convert the null-delimited buffer of strings returned from registry values of
+     * type {@link WinNT.REG_MULTI_SZ} to an array of strings.
+     *
+     * @param data
+     *            A buffer containing strings delimited by a null character, ending
+     *            with two null characters.
+     * @return An array of strings corresponding to the strings in the buffer.
+     */
+    public static String[] regMultiSzBufferToStringArray(Memory data) {
```

Comment:
Would you mind making this package private? While I see nothing obviously wrong, I'd like this not to be exposed as API.

Reply:
Makes sense